android/app/src/main/java/tuerantuer/app/integreat/MainActivity.java

```diff
@@ -3,17 +3,68 @@
 import android.content.res.Configuration;
 import android.os.Bundle;
 import com.facebook.react.ReactActivity;
+import android.content.SharedPreferences;
+import android.preference.PreferenceManager;
 
+import java.io.File;
 import java.util.Locale;
 
+
 public class MainActivity extends ReactActivity {
 
     private Locale currentLocale;
 
+    private void clearCachedContent()
+    {
+        // delete all files
+        File dir = getApplicationContext().getFilesDir();
+        if (dir.isDirectory())
+        {
+            String[] children = dir.list();
+            for (int i = 0; i < children.length; i++)
+            {
+                if (children[i].endsWith(".json")) {
```

Comment:
I think we can feel adventurous here and just remove all the files. I'm a bit afraid that some very old version put some files here which we would not expect.

Reply:
Yeah, I also don't see a reason why we shouldn't just  delete  everything.

Reply:
Don't think that's necessary, json-Files, pdf-Files, etc. are the only files that xamarin creates.

Reply:
Hm I'd still favor to just delete everything recursively just in case one of the 40 android versions created a file which we are not aware of.

Reply:
Yeah it is probably not necessary, but I also think it just does not hurt and would be better if it should happen that there are different files (also I don't think there are any other files)